### PR TITLE
Add details for floats and strings for user attributes object

### PR DESCRIPTION
**Pull request by lydia-xie**
### Summary

My PR...

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by shihanxiong — APPROVED** [review]

File: _docs/_api/objects_filters/user_attributes_object.md [hunk 1]
```diff
@@ -85,9 +85,9 @@ The following data types can be stored as a custom attribute:
...
-| Floats |  |
+| Floats | Float custom attributes are numbers with a decimal point. For example, you can use floats to store user ratings for products or services. |
 | Integers | Integer custom attributes may be incremented by positive or negative integers by assigning them an object with the field "inc" and the value by which you want to increment them. <br><br>Example: `"my_custom_attribute_2" : {"inc" : int_value},`|
-| Strings |  |
+| Strings | String custom attributes are sequences of characters used to store text data. For example, you can use strings to store user names, email addresses, or preferences. |
```
Comment (lydia-xie) [inline _docs/_api/objects_filters/user_attributes_object.md:90]:
```suggestion
| Strings | String custom attributes are sequences of characters used to store text data. For example, you can use strings to store first and last names, email addresses, or preferences. |
```

File: _docs/_api/objects_filters/user_attributes_object.md [hunk 2]
```diff
@@ -85,9 +85,9 @@ The following data types can be stored as a custom attribute:
...
 | Booleans | `true` or `false` |
 | Dates | Must be stored in the [ISO 8601][19] format or in any of the following formats: <br>- `yyyy-MM-ddTHH:mm:ss:SSSZ` <br>- `yyyy-MM-ddTHH:mm:ss` <br>- `yyyy-MM-dd HH:mm:ss` <br>- `yyyy-MM-dd` <br>- `MM/dd/yyyy` <br>- `ddd MM dd HH:mm:ss.TZD YYYY` <br><br>Note that "T" is a time designator, not a placeholder, and should not be changed or removed. <br><br>Time attributes without a time zone will default to Midnight UTC (and will be formatted on the dashboard as the equivalent of Midnight UTC in the company's time zone). <br><br> Events with timestamps in the future will default to the current time. <br><br> For regular custom attributes, if the year is less than 0 or greater than 3000, Braze stores these values as strings on the user. |
-| Floats |  |
+| Floats | Float custom attributes are numbers with a decimal point. For example, you can use floats to store user ratings for products or services. |
```
Comment (internetisaiah) [inline _docs/_api/objects_filters/user_attributes_object.md:88]: Couple things to specific here:

1. Can you assign a negative number? Or must it always be positive?
2. I think its good to add some info on the general "precise-ness" of the float. For context, in many program languages there are floats and doubles. For example in C++ a float holds up to 4 bytes while a double holds up to 8 bytes of memory (i.e. approximately 7 decimal digits of precision for floats vs 15 for doubles). That means if you want to reference an integer like `1.29837918769`, you will want to use a float over a double. We don't need to be _overly_ precise here, but maybe giving a general "up to 5 decimal digits" or something is fine (once we verify with prod/eng).

**Review by internetisaiah — CHANGES_REQUESTED** [review]
Quick follow up question/statements here before merging.

File: _docs/_api/objects_filters/user_attributes_object.md (hunk 2 above)
Comment (lydia-xie) [reply _docs/_api/objects_filters/user_attributes_object.md:88]: 1. I believe you can assign negative and positive numbers for floats. So for cases where a custom attribute is like a bank account balance, the float could be negative.
2. I don't think we have a specific limit for the number of digits in a float. I'm not entirely sure about the guidance here for how many decimal digits are recommended.

@shihanxiong - do you have any additional context here for @internetisaiah 's questions?

Comment (shihanxiong) [reply _docs/_api/objects_filters/user_attributes_object.md:88]: 1. float can be a negative number
2. Afaik ruby has a limit of 15 digits to float type

Comment (lydia-xie) [inline _docs/_api/objects_filters/user_attributes_object.md:88]:
```suggestion
| Floats | Float custom attributes are positive or negative numbers with a decimal point. For example, you can use floats to store account balances or user ratings for products or services. |
```

Comment (internetisaiah) [reply _docs/_api/objects_filters/user_attributes_object.md]: cool. given that the max value is so high, i doubt customers will ever exceed it, so i don't think we need to specify that personally. but i trust your judgement @lydia-xie 👍🏽

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**